### ISSUE TITLE
Loading new rows with data attributes 

### ISSUE THREAD

**Issue by TomasJerrySebo** [fooplugins/FooTable#748]
Is it possible to load new rows with rows.load() -
http://fooplugins.github.io/FooTable/docs/examples/advanced/loading-rows.html 

So the result of the row looks like follows: ?
`<tr data-id="9180741188" data-name="RAN Sharing 3G Data Flow Diagram" data-description="The data flow diagram explains how the data is being transformed by various modules in the program that is used by particular operator for Radio Netw." data-price="3101" data-status="30008" data-type="104" data-image="https://syntheticspheres.box.com/shared/static/dmrexul8pfe0l31qut1gpjd0wjqmr3ju.png" class="row diagram-row"><td data-id="9180741188" class="diagram-expand"><img id="remove-diagram" class="removeButton" style="width:15px;" src="//cdn.shopify.com/s/files/1/0255/1589/t/7/assets/remove.png"></td><td>9180741188</td><td style="display: table-cell;">RAN Sharing 3G Data Flow Diagram</td><td class="diagram-expand">2016-10-11 21:42:10</td><td>NO AUTHOR</td></tr>`

So basically I need to add different data attributes to different elements of the row. If not could someone suggest a workaround? Thanks

**Comment by AlanGreyjoy** [comment]
Have you found a work around for this?

**Comment by AlanGreyjoy** [comment]
Since this is a a super old issue, and google brought me here, the work around is the get your data, then run it through a foreach loop and append the new row to your table.

If you use $.get you can put the loop in the .then(function()) and then setup the table in the .done(function()) of the $.get.
